app/cypress/support/index.ts

```diff
@@ -17,11 +17,27 @@
 // Import commands.js using ES2015 syntax:
 import './commands'
 
+
 // Alternatively you can use CommonJS syntax:
 // require('./commands')
 
+const cachedBackendResponses = {}
+const hashCode = (s) => s.split('').reduce((a, b) => {a = ((a << 5) - a) + b.charCodeAt(0); return a & a}, 0)
+
 // run before each test in each file
 beforeEach('Logs in via UI', () => {
+  cy.intercept({url: /^https:\/\/localhost:3000\/api/}, (req) => {
```

Comment:
this url seems kinda restrictive if we wanted to change anything. I think a substring would be sufficient